### DISCUSSION TITLE
Best SEO practices on Joomla “asteroid" site?

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#235] [Q&A] [answered]
Hello

Does any one have some suggestions for the best SEO practices on my Joomla “asteroid" site.

This is how my site is built:

- I have 7 menus on my site each going to a “featured article page.”
- There are NO articles on the site what so ever = "featured articles page" is a place holder.
- Instead I have 7 modules (Custom JCE Modules with layout options) going to the 7 menu items.

First:

- In Global Settings/Site/SEO settings I have things set up as in the screen shot

a) The site name is my name

Second I would like to add:

b) Page Title
c) Description
d) Keywords

b)

- On each of the 7 pages in the “Page Display” section I can add the option to add “Title”

a+b)

- Have set up page titles for all 7 pages and can see them working with the Site name.

c)

- I guess that the “Description” goes in the Meta Description section?

d)

- But where do the Keywords go?

Am I missing something? Hope this makes sense? Grateful for any tips?

thanks

Paul

<img width="960" alt="01-SEO-Settings" src="https://user-images.githubusercontent.com/9163424/198239718-6b2d180e-dd89-4da2-8b6a-475a2ef4cbbb.png">

<img width="960" alt="02-PAge-Display" src="https://user-images.githubusercontent.com/9163424/198239772-bedbf2ee-e521-45e2-95ef-991689727485.png">

<img width="960" alt="03-Meta-Data" src="https://user-images.githubusercontent.com/9163424/198239907-863e1d0d-120f-459e-b599-c06bcc6e6a44.png">

**Comment by Chacapamac** [comment, marked as answer]
If you have no articles, you can place your SEO at the **Joomla Menu level** 

**- Page Title
- Description
- URL**
![title](https://user-images.githubusercontent.com/1907276/198630253-423be83a-e87f-4d3a-b0d5-b5fb2554b712.jpg)

![description](https://user-images.githubusercontent.com/1907276/198632268-203a3ad0-bfd2-4ab1-b0e8-c12e36b62c5e.jpg)

Also in “Menu Edit Item” you can add some keywords there
![alias](https://user-images.githubusercontent.com/1907276/198632355-d2b36be9-dabf-48b1-86ae-c9ac6cef29a8.jpg)

**You can use this tool to help you build a correct Title/Description:** https://www.highervisibility.com/seo/tools/serp-snippet-optimizer/

**After in your text:in each Page/Menus**
**• Add Keywords in Headers in order of Importance.**
— H1 for a UNIQUE page Title
— H2 for IMPORTANT Sub sections in your pages
— H3 for Paragraphs
— H4 going down if possible 

**• Add Keywords in Images.**
— Directly in image file name e.g. “my-seo-image.jpg”
— In “Alt” tag add small description/keywords
— In Captions, if possible

**Also really important to maximize compression of images.**
Use this Google tools —> https://squoosh.app/

**• Add Keywords in your text.**

**Google will do the rest**

**Comment by paurray** [comment]
Hello @Chacapamac 

it has been a long bank holiday weekend here.
Many thanks for the detailed tips.
Gratefully appreciated.

checking them out now

more later, thanks

Paul

**Comment by paurray** [comment]
@Chacapamac 

Have been playing with this and obviously it all works exactly as expected. 
Thanks posting those bonus tool links.

Greatly appreciated

all the best Paul

**Comment by Chacapamac** [comment]
Happy to Help Paul.
Good Day!
